### ISSUE TITLE
Ekuiper manager rule edit page bug

### ISSUE THREAD

**Issue by QBH-insist** [lf-edge/ekuiper#1774]
**Environment**:

- eKuiper&manager version: 1.9.0 

**What happened and what you expected to happen:**
The rule edit page displays the options tab normally

**How to reproduce it :**

- Use the Post http://host:port/rules API to create a rule
- The rule edit page doesn't display the options tab until a submission is made once
![image](https://user-images.githubusercontent.com/39401478/231926166-d6e1bb4f-15fb-48e8-a0d1-7bc33ff5dea2.png)

**Comment by superrxan** [comment]
@QBH-insist  Could you please share with us your http post body ?

**Comment by QBH-insist** [comment]
> @QBH-insist Could you please share with us your http post body ?

```
{
  "id": "test-upload-rule",
  "sql": "SELECT * FROM test_stream",
  "actions": [{
    "rest": {
        "url": "http://ip/upload/event",
        "method": "POST",
        "insecureSkipVerify": true,
        "bodyType": "json",
        "headers": {
            "Content-Type": "application/json"
        },
        "sendSingle": true,
        "concurrency": 2,
        "cleanCacheAtStop": false
      }
  }]
}
```

**Comment by ngjaying** [comment]
Fixed in eKuiper manager 1.9.1
